pkgs/development/python-modules/lmfit/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, buildPythonPackage, fetchPypi,
+  setuptools-scm, asteval, numpy, scipy, uncertainties,
+  pytest-cov, pytestCheckHook, pandas, matplotlib,
+}:
+
+buildPythonPackage rec {
+  pname = "lmfit";
+  version = "1.2.1";
+
+  format = "setuptools";
```

Comment:
Slight preference towards pyproject build, as it is more future-proof.

Requires `setuptools` and `setuptools-scm` in `nativeBuildInputs`. Not `wheel`, putting it in there is cargo culting.